tests/test_direct_api.py

```diff
@@ -2556,6 +2556,17 @@ def test_plane_init(self):
         with self.assertRaises(TypeError):
             Plane(Edge.make_line((0, 0), (0, 1)))
 
+        # can be instantiated from planar faces of different surface types
```

Comment:
Tests must make assertions to validate the results and not just run to completion.

Reply:
I've extended the test to also verify properties of the two trapezoid faces, which are created as Geom_BSplineSurface by loft() and are the focus of the introduced changes.